tests/integration/test_dynamodb.py

```diff
@@ -427,29 +427,50 @@ def test_return_values_in_put_item(self, dynamodb):
 
         # items which are being used to put in the table
         item1 = {PARTITION_KEY: "id1", "data": "foobar"}
+        item1b = {PARTITION_KEY: "id1", "data": "barfoo"}
         item2 = {PARTITION_KEY: "id2", "data": "foobar"}
 
         response = table.put_item(Item=item1, ReturnValues="ALL_OLD")
         # there is no data present in the table already so even if return values
         # is set to 'ALL_OLD' as there is no data it will not return any data.
-        assert not response.get("Attributes")
+        assert response["ResponseMetadata"]["HTTPStatusCode"] == 200
```

Comment:
nitpick: we are calling the same asserts a bunch of times. To be a bit more concise, you can wrap them in an inner function.